tools/android/json_worker_wrapper.py

```diff
@@ -0,0 +1,55 @@
+from io import StringIO
+import json
+import select
+import os
+import sys
+import traceback
+
+
+def wait_for_stdin_json():
+  stdin = open("/dev/stdin", "rb")
```

Comment:
Does this work on Windows and MacOS? I'm pretty sure that Python has built-in convenience abstractions for stdin/stdout/stderr access, right?